src/Bicep.LangServer.UnitTests/Snippets/SnippetsProviderTests.cs

```diff
@@ -396,6 +396,10 @@ public void GetNestedResourceDeclarationSnippets_WithChildResources_ShouldReturn
                     x.Prefix.Should().Be("res-automation-job-schedule");
                 },
                 x =>
+                {
+                    x.Prefix.Should().Be("res-automation-module");
+                },
```

Comment:
This change is coming from version change correct?

Reply:
Yes.